WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/widget/ViewsWidgetUpdater.kt

```diff
@@ -92,6 +93,7 @@ class ViewsWidgetUpdater
         listIntent.putExtra(AppWidgetManager.EXTRA_APPWIDGET_ID, appWidgetId)
         listIntent.putExtra(SHOW_CHANGE_VALUE_KEY, showChangeColumn)
         listIntent.putExtra(COLOR_MODE_KEY, colorModeId)
+        listIntent.putExtra(VIEW_TYPE_KEY, ViewType.WEEK_VIEWS.ordinal)
```

Comment:
I'm not really sure about using ordinals. Enums are serializable, so I wonder maybe we can pass them as-is?

Reply:
I'm not sure either :/, I tried passing them as serializables and it didn't work but I can try again.